server/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "server"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+zbus = "3.14.1"
+tokio = { version = "1", features = ["full"] } # look for specific features
+oo7 = { path = "../client", features = ["unstable"] }
+serde = "1.0"
+once_cell = "1.19.0"
```

Comment:
Avoid using once_cell, we are trying to get rid of it as what we usually need it for can be covered with std nowadays

Reply:
Done, thanks!